README.md

```diff
@@ -249,29 +256,29 @@ This returns a gateway that will:
    knows where to send outbound RPCs).
 
 You can customize the above behavior by setting various fields on the gateway:
-* `Client`: The transport to use for outbound RPCs. (This can also be overridden
+
+- `Client`: The transport to use for outbound RPCs. (This can also be overridden
   on a per-service basis, if some services require different middleware, such as
   auth, than others).
-* `ClientOptions`: The Connect client options to use for outbound RPCs. This
+- `ClientOptions`: The Connect client options to use for outbound RPCs. This
   allows customizing things like interceptors and protocols. If some backends
   only support gRPC, you can configure that with a client option.
-* `MaxParallelismPerRequest`: The concurrency limit for handling a single Knit
+- `MaxParallelismPerRequest`: The concurrency limit for handling a single Knit
   request. Note that this controls the parallelism of issuing entry-point RPCs
   and the parallelism of invoking resolvers. This setting cannot be enforced
   inside of resolver implementations: if a resolver implementation starts other
   goroutines to operate with additional parallelism, this limit may be exceeded.
-* `Route`: This is a default route. If you have one application that will
-  receive most (or all) of the Connect/gRPC traffic, configure it here. Then you 
+- `Route`: This is a default route. If you have one application that will
+  receive most (or all) of the Connect/gRPC traffic, configure it here. Then you
   only need to include routing information when registering services that should
   be routed elsewhere.
-* `TypeResolver`: This is an advanced option that is usually only useful or
+- `TypeResolver`: This is an advanced option that is usually only useful or
   necessary when using dynamic RPC schemas. This resolver provides descriptors
   for extensions and messages, in case any requests or responses include
   extensions or `google.protobuf.Any` messages.
 
 > NOTE: If you want to configure a custom codec for outbound RPCs, to customize
-> content encoding, you must use `knit.WithCodec` **instead of**
-> `connect.WithCodec` when creating the Connect client option.
+> content encoding, you must use `knit.WithCodec` **instead of** > `connect.WithCodec` when creating the Connect client option.
```

Comment:
This looks like a typo.
```suggestion
> content encoding, you must use `knit.WithCodec` **instead of**
> `connect.WithCodec` when creating the Connect client option.
```